scripts/local_rabbitmq.sh

```diff
@@ -9,7 +9,7 @@ RABBITMQ_LEAGUE_SERVICE_USER=faf-league-service
 RABBITMQ_LEAGUE_SERVICE_PASS=banana
 RABBITMQ_LEAGUE_SERVICE_VHOST=/faf-lobby
 
-docker run -d -p 5672:5672 --restart unless-stopped --name faf-rabbitmq rabbitmq:3.8-management-alpine
+docker run -d -p 5672:5672 --restart unless-stopped --name faf-rabbitmq rabbitmq:3.9-alpine
```

Comment:
I have had issues with switching to rabbitmq 3.9 in the past, but it looks like at some point I tried it and it just worked fine. So I can't really offer any advice unfortunately. Maybe it's more of an issue with the dev setup than the rabbitmq version itself?

https://github.com/FAForever/server/pull/944

Reply:
Are you referring to the messages lingering during tests?

Reply:
I'm not sure if it's worth the trouble to fix that. I fixed the test with a workaround without compromising the meaningfulness of the test I think

Reply:
I think you also sent me a message on Zulip that you hadn't gotten rabbitmq to work. If everything is working then that's great!

Reply:
That was just me being confused, because there are some tests that deliberatly test failure modes when starting rabbitmq the wrong way and due to a dependency change I had to slightly refactor how the errors get thrown. From glancing at the failing tests I first thought that it failed because rabbitmq had a weird problem until I understood that the whole point of the test was to make rabbitmq error out.